tests/integration/store/test_pandas_store.py

```diff
@@ -797,3 +796,27 @@ def test_daterange_fails_with_timezone_start(library):
     with pytest.raises(ValueError):
         library.read('MYARR', date_range=DateRange(start=dt(2015, 1, 1, tzinfo=mktz())))
 
+def test_data_info_series(library):
```

Comment:
For these new `data_info` tests, it would be good to assert the shape and content of the data_info property